Gpiokmdfdemo/asl/dragon.asl

```diff
@@ -0,0 +1,58 @@
+//

```

Comment:
There are .asl files for the dragonboard, the rpi and the MinnowBoardMax, but I don't see one for the UP Board.  Does it need it?

Reply:
It not necessary to create a asl file for Up board, as on compilation is generates .inf, .sys, .cat files and we could successfully install the GPIOKmdf drivers on UP board